package.json

```diff
@@ -66,18 +66,18 @@
   },
   "dependencies": {
     "cids": "^1.0.0",
-    "class-is": "^1.1.0",
     "multicodec": "^2.0.0",
     "multihashing-async": "^2.0.0",
-    "protons": "^2.0.0",
+    "protobufjs": "^6.10.2",
```

Comment:
I would've prefered https://github.com/mapbox/pbf, but I guess that ship has sailed.

Reply:
From memory it doesn't support gRPC service definitions?

Reply:
That can well be.